MdePkg/Test/MdePkgHostTest.dsc

```diff
@@ -57,3 +57,4 @@
   MdePkg/Test/Mock/Library/GoogleTest/MockMemoryAllocationLib/MockMemoryAllocationLib.inf
   MdePkg/Test/Mock/Library/GoogleTest/MockPciExpressLib/MockPciExpressLib.inf
   MdePkg/Test/Mock/Library/GoogleTest/MockUefiDevicePathLib/MockUefiDevicePathLib.inf
+  MdePkg/Test/Mock/Library/GoogleTest/MockPerformanceLib/MockPerformanceLib.inf

```

Comment:
```suggestion
  MdePkg/Test/Mock/Library/GoogleTest/MockUefiDevicePathLib/MockUefiDevicePathLib.inf
MdePkg/Test/Mock/Library/GoogleTest/MockPerformanceLib/MockPerformanceLib.inf
```

Reply:
I'm not sure what to change.
Do I need to remove the white space which in front of inf ?

Reply:
Line 59 has an unneeded whitespace. The 'M' on line 59 should line up with the 'M' on line 60